src/App.tsx

```diff
@@ -1,109 +1,37 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
+
+function getUser(userId: number): User | null {
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/233081456-926e3424-c0a1-4ad9-8c9b-d9d33ff29c97.png)
```suggestion
function getUserById(id: number): User | null {
```